project/Dependencies.scala

```diff
@@ -26,13 +26,13 @@ object Dependencies {
   val logbackVersion = "1.2.3"
 
   val scala212Version = "2.12.11"
-  val scala213Version = "2.13.1"
+  val scala213Version = "2.13.3"
 
   val reactiveStreamsVersion = "1.0.3"
 
-  val sslConfigVersion = "0.4.1"
+  val sslConfigVersion = "0.4.2"
 
-  val scalaTestVersion = "3.1.1"
+  val scalaTestVersion = "3.2.0"
```

Comment:
This change has some downstream impact. I thought we discussed this before, but can't find it and am not sure what conclusion we landed on (if any). Opened #29368 for that.

Reply:
I have removed the commits that update `scalatest, mockito, ssl-config, sbt` from this PR